plugin/build.gradle.kts

```diff
@@ -107,6 +107,10 @@ pluginBundle {
     tags = listOf("ktlint", "kotlin", "linting")
 
     (plugins) {
+        "ktlintBasePlugin" {
+            id = "org.jlleitschuh.gradle.ktlint-base"
+            displayName = "Ktlint Gradle Base Plugin"
+        }
```

Comment:
I was hoping that we could avoid publishing this as it's own independent artifact. But sure, if this is what we have to do.

Is there any way we can not do this?